weave

```diff
@@ -115,10 +115,35 @@ docker_run_options() {
     echo --privileged --net=host $(docker_sock_options)
 }
 
+docker_exec_cni_options() {
```

Comment:
The naming is unfortunate, especially given the existence of `docker_run_options`, since both `run` and `exec` are docker commands, and `docker_run_options` is named after the docker command but `docker_exec_cni_options` is not.